l10nzhtw.properties

```diff
@@ -0,0 +1,14 @@
+category=Localization
+description=SonarQube Traditional Chinese Pack
+homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zhtw
+archivedVersions=
+publicVersions=1.0
+
+defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
+defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+
+1.0.description=Traditional Chinese Pack Support SonarQube 8.0
+1.0.sqVersions=[7.9,LATEST]
```

Comment:
Your description says 8.0 and your compatibility says 7.9+. Please make them match. :-)

& sorry I didn't notice this before.